### PR TITLE
Fixed IE issues

### PR DESCRIPTION
**Pull request by urkaver**
1 Replaced element.remove() with element.parentElement.removeChild because IE doesn't support it.
2 Replaced window.scrollX with window.pageXOffset because IE doesn't support it.

### CODE REVIEW

File: angularjs-datetime-picker.js [hunk 1]
```diff
@@ -34,7 +34,7 @@
     var datetimePickerEl;
     var _this = this;
     var removeEl = function(el) {
-      el && el.remove();
+      el && el.parentElement.removeChild(el);
```
Comment (subramaniam-s17) [inline angularjs-datetime-picker.js:37]: Why is this change required?

Comment (urkaver) [reply angularjs-datetime-picker.js:37]: Because IE doesn't recognize function remove and throw error with message "Object doesn't support property or method 'remove'". More details here: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

**Review by subramaniam-s17 — APPROVED** [review]
